src/App/screens/Instructor/components/Nav/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import Link from 'react-router/Link'
+import Logo from '../Logo'
+import NavLink from './components/NavLink'
+
+class Nav extends React.Component {
+  
+  state = {
+    isOpen: false,
+  }
+
+  toggle() {
+    this.setState({
+      isOpen: !this.state.isOpen,
+    })
+  }
+
+  render() {
+    const {pathname, routes} = this.props
+
+    return (
+      <header className='pv3 pv0-ns bg-black-70'>
+
+        <div className='mw8 center relative flex-ns items-center-ns'>
+
+          <div className='ph4 flex items-center justify-between'>
+            <div className='flex items-center'>
+              <div className='w2 mr2'>
+                <Logo />
+              </div>
+              <div className='white'>
+                Instructor Center
+              </div>
+            </div>
+          </div>
+
+          <nav className={`
+            mt3 mt0-ns
+            ${this.state.isOpen
+              ? 'flex'
+              : 'dn'
```

Comment:
Normally I just wouldn't render elements, but since Tachyons is not in JS, I relied on `display: none` for the mobile menu collapse. Could use JS screen size instead to apply classes, but I'll give the `*-ns` approach a little while longer of a test to see how it scales.